frontend/src/components/Map/hooks/usePopupResult.ts

```diff
@@ -5,8 +5,11 @@ import { PopupResult } from 'modules/trekResult/interface';
 import { getTouristicContentPopupResult } from 'modules/touristicContent/connector';
 import { useRouter } from 'next/router';
 import { getDefaultLanguage } from 'modules/header/utills';
-import { getOutdoorSitePopupResult } from '../../../modules/outdoorSite/connector';
+import { CommonDictionaries } from 'modules/dictionaries/interface';
+import { getCommonDictionaries } from 'modules/dictionaries/connector';
+import { ONE_DAY } from 'services/constants/staleTime';
```

Comment:
Maybe create a HALF_DAY constant? It's used in a LOT of files

Reply:
With the DRY, it's now call once. Do you think it's still necessary?

Reply:
Nope, perfect this way 👌